src/napari_spatialdata/_interactive.py

```diff
@@ -27,6 +28,27 @@ class Interactive:
     None
     """
 
+    def add_element(self, coordinate_system_name: str, element: str) -> SpatialData:
+        for element_type, element_name, _ in self._sdata.filter_by_coordinate_system(
+            coordinate_system_name
+        )._gen_elements():
+            if element_name == element:
+                if element_type == "images":
+                    self._sdata_widget.viewer_model.add_sdata_image(self._sdata, coordinate_system_name, element)
+                elif element_type == "labels":
+                    self._sdata_widget.viewer_model.add_sdata_labels(self._sdata, coordinate_system_name, element)
+                elif element_type == "points":
+                    self._sdata_widget.viewer_model.add_sdata_points(self._sdata, coordinate_system_name, element)
+                elif element_type == "shapes":
+                    if type(self._sdata.shapes[element].iloc[0][0]) == shapely.geometry.point.Point:
+                        self._sdata_widget.viewer_model.add_sdata_circles(self._sdata, coordinate_system_name, element)
+                    elif (type(self._sdata.shapes[element].iloc[0][0]) == shapely.geometry.polygon.Polygon) or (
+                        type(self._sdata.shapes[element].iloc[0][0]) == shapely.geometry.multipolygon.MultiPolygon
+                    ):
+                        self._sdata_widget.viewer_model.add_sdata_shapes(self._sdata, coordinate_system_name, element)
+
+                break
+
```

Comment:
thinking better about this, I think it's not good that we are duplicating the logic of _sdata_widgets.py/_add_shapes(), 

I would remove this code and the corresponding code from _add_shapes(); put it in a function; and call that function from here and from _add_shapes(). The function should tell if we are dealing with a point or with a polygon/multipolygon. It could for instance returnt the type directly
```suggestion
            if element_name == element:
                if element_type == "images":
                    self._sdata_widget.viewer_model.add_sdata_image(self._sdata, coordinate_system_name, element)
                elif element_type == "labels":
                    self._sdata_widget.viewer_model.add_sdata_labels(self._sdata, coordinate_system_name, element)
                elif element_type == "points":
                    self._sdata_widget.viewer_model.add_sdata_points(self._sdata, coordinate_system_name, element)
                elif element_type == "shapes":
                    if type(self._sdata.shapes[element].iloc[0][0]) == shapely.geometry.point.Point:
                        self._sdata_widget.viewer_model.add_sdata_circles(self._sdata, coordinate_system_name, element)
                    elif (type(self._sdata.shapes[element].iloc[0][0]) == shapely.geometry.polygon.Polygon) or (
                        type(self._sdata.shapes[element].iloc[0][0]) == shapely.geometry.multipolygon.MultiPolygon
                    ):
                        self._sdata_widget.viewer_model.add_sdata_shapes(self._sdata, coordinate_system_name, element)

                break

```